source/how-tos/enable-intel-tsn.rst

```diff
@@ -0,0 +1,311 @@
+.. _tsn:
+
+TSN
+~~~~~~~~~~~~~~~~~
+
+Overview
+--------------------
+
+TSN (Time Sensitive Networking) solution is a set of capabilities that offers Time Synchronization, Traffic Shaping, set of IEEE open standard communication by bringing industrial-grade robustness and reliability to Ethernet . TSN removes the need for physical separation of critical and non-critical communication networks, thus allowing open data exchange between operations and enterprise - a concept at the heart of the Industrial Internel of Things (IIoT).
+To learn more about TSN, see |TSN| for details.
+
+.. |TSN| raw:: html
+
+   <a href="https://www.intel.com/content/www/us/en/industrial-automation/programmable/applications/automation/tsn.html" target="_blank">TSN</a>
+
+
+.. _tsn_container:
+
+TSN container
+--------------------
+
+Container support is a generic SBL feature. Please refer to |Container| and |Container Security| for generic details.
+
+.. |Container| raw:: html
+
+   <a href="https://slimbootloader.github.io/tools/GenContainer.html" target="_blank">Container</a>
+
+.. |Container Security| raw:: html
+
+   <a href="https://slimbootloader.github.io/security/container-security.html?highlight=container#container-security" target="_blank">Container Security</a>
+
+
+SBL defines a container called IPFW for settings related to TSN and TSN is managed as a component inside this IPFW container.

```

Comment:
SBL defines a container called IPFW for settings related to various silicon IP and settings related to TSN are managed as a component inside this IPFW container.